### DISCUSSION TITLE
Add the app to homebrew

### DISCUSSION THREAD

**Discussion by JimmyHoenderdaal** [rxhanson/Charmstone-Community#31] [Ideas]
I'd like to try the app out but I don't want to create an Apple account. Either make it available on homebrew or give us an installer we can download.

**Comment by rxhanson** [comment]
Thanks for the feedback. This is on my backlog, and I just haven't gotten to it yet.

**Comment by alexthehurst** [comment]
+1 - I bought Charmstone on another computer, but would like to be able to use it on my work computer which has the Mac app store disabled.

Thanks for the really neat tool!
